src/scripts/components/main/language-selector/language-selector.tsx

```diff
@@ -21,6 +21,14 @@ const LanguageSelector: FunctionComponent<Props> = ({className = ''}) => {
   const selectedLanguage = useSelector(languageSelector);
   const dispatch = useDispatch();
 
+  const setLanguage = (language: Language) => {
+    localStorage.setItem('language', language);
+
+    if (localStorage.getItem('language')) {
```

Comment:
what is this check for? if you set the item first and then check if it's there it should be always true